docs/claudie-workflow/claudie-workflow.md

```diff
@@ -54,56 +61,64 @@ Context box is Claudie's "control unit". It holds pending configs, which need to
   // Deletes the config from the database.
   rpc DeleteConfigFromDB(DeleteConfigRequest) returns (DeleteConfigResponse);
 ```
+
 ### Flow
+
 - Receives a `config` from Frontend, calculates its `msChecksum` and saves it to the database
 - Periodically checks for `config` changes and pushes the `config` to the `schedulerQueue` if `msChecksum` != `dsChecksum`
 - Periodically checks for `config` changes and pushes the `config` to the `builderQueue` if `dsChecksum` != `csChecksum`
 - Receives a `config` with the `desiredState` from Scheduler and saves it to the database
 - Receives a `config` with the `currentState` from Builder and saves it to the database
 
 ### Variables used
+
 | variable     | meaning                |
 | ------------ | ---------------------- |
 | `msChecksum` | manifest checksum      |
 | `dsChecksum` | desired state checksum |
 | `csChecksum` | current state checksum |
 
-
 ## Scheduler
+
 Scheduler brings the infrastructure to a desired the state based on the manifest contained in the config that is received from Context-box.
 
 Scheduler also monitors the health of current infrastructure and manages any operations based on actual health state (e.g. replacement of broken nodes, etc. *[work in progress]*).
 
 ### API
-```
+
+```sh
```

Comment:
Doesn't look like a shell command to me :)

Reply:
Markdown linter did not like a code block without specified formatting. I just used a `sh` to silence the warnings, can remove them if we want.

Reply:
Ah, I see. However I wouldn't put this sentence into a code-block.